sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/JsonExpressionsSuite.scala

```diff
@@ -402,13 +402,13 @@ class JsonExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper with
     val schema = StructType(StructField("a", IntegerType) :: Nil)
     checkEvaluation(
       JsonToStructs(schema, Map.empty, Literal(jsonData), gmtId),
-      null
+      InternalRow(null)
     )
 
     // Other modes should still return `null`.
     checkEvaluation(
       JsonToStructs(schema, Map("mode" -> PermissiveMode.name), Literal(jsonData), gmtId),
```

Comment:
The default mode is `PermissiveMode` now, so this looks necessary to change.

Reply:
Changed to `FailFastMode.name`